oci-unit-tests/helper/test_helper.sh

```diff
@@ -15,15 +15,51 @@ load_shunit2() {
 
 debug() {
     if [ -n "${DEBUG_TESTS}" ]; then
-        if [ "${1}" = "-n" ]; then
-            shift
-            echo -n "$@"
-        else
-            echo "$@"
-        fi
+	if [ "${1}" = "-n" ]; then
+	    shift
+	    echo -n "$@"
+	else
+	    echo "$@"
+	fi
     fi
 }
 
+# Creates a new docker container from $DOCKER_IMAGE.
+#
+# By default, creates a temporary container running the software under
+# test as a daemon service.  The container is automatically removed
+# when the daemon exits.  All parameters to run_container_server() are
+# passed directly through to the contained software daemon.  The docker
+# behavior can be adjusted through environmental variables listed
+# below.  In particular, $suffix can be used to distinguish between
+# multiple containers used in a given test, such as 'client' and
+# 'server'.
```

Comment:
Because containers can be either servers of clients, WDYT about renaming this function to something more generic like `run_container_application` or `run_container_service`?

Reply:
Yes, those both sound better.  I think since this routine specifically runs daemon services (ala the -d flag), "run_container_service" is probably the more intuitive name.  I'll rename.